.releaserc

```diff
@@ -17,19 +17,9 @@
         "message": "chore(release): ${nextRelease.version}"
       }
     ],
-    [
-      "@semantic-release/commit-analyzer",
-      {
```

Comment:
why did you remove these?

Reply:
we wanted to be able to make code changes that do not affect the business logic. e.g. updating the `.snyk` file. Following the commit standards, this would be considered a `chore`, however it should not lead to a release. The releaseRules meant that it would lead to a release.

We may have been overzealous in removing it all, but I think `docs, chore, style` shouldn't require patch versions at least.

Reply:
Ok :)